pkg/kyma/cmd/test/run/cmd.go

```diff
@@ -0,0 +1,147 @@
+package run
+
+import (
+	"fmt"
+	"math/rand"
+	"strings"
+	"time"
+
+	oct "github.com/kyma-incubator/octopus/pkg/apis/testing/v1alpha1"
+	"github.com/kyma-project/cli/internal/kube"
+	"github.com/kyma-project/cli/pkg/api/octopus"
+	"github.com/kyma-project/cli/pkg/kyma/cmd/test"
+	"github.com/kyma-project/cli/pkg/kyma/core"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+type command struct {
+	opts *options
+	core.Command
+}
+
+func NewCmd(o *options) *cobra.Command {
+	cmd := command{
+		Command: core.Command{Options: o.Options},
+		opts:    o,
+	}
+
+	cobraCmd := &cobra.Command{
+		Use:     "run",
+		Short:   "Run tests on a running Kyma cluster",
+		Long:    `Run tests on a running Kyma cluster`,
+		RunE:    func(_ *cobra.Command, _ []string) error { return cmd.Run() },
+		Aliases: []string{"r"},
+	}
+
+	cobraCmd.Flags().StringVarP(&o.Name, "name", "n", "", "Name for the new test suite")
+	cobraCmd.Flags().StringVarP(&o.Tests, "tests", "s", "", "Test names to execute. Example: --tests=cluster-users-test,test-api-controller-acceptance. Overlaps with --all")
+	cobraCmd.Flags().BoolVarP(&o.Wait, "wait", "w", false, "Wait for test execution to finish")
+	cobraCmd.Flags().IntVarP(&o.Timeout, "timeout", "t", 120, "Timeout for test execution (in seconds)")
```

Comment:
This flag should be a `time.Duration` to be consistent with the install and uninstall commands.

Reply:
Fixed